### PR TITLE
Added yml file config as a source

### PR DESCRIPTION
**Pull request by muhfajar**
I just added yml as a config source, is that ok?

### CODE REVIEW

File: wasserstoff/utils.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from collections import Mapping
 import json
+import yaml
```
Comment (lk-geimfari) [inline wasserstoff/utils.py]: It's should be optional. Add it like that:

```python
try:
    import yaml
except ImportError:
    raise ImportError("You should install pyyaml.")
```
i.e library can normally work without requirements, but if someone need yaml then he can just install pyyaml and use it. Do you agree?

File: setup.py [hunk 2]
```diff
@@ -46,6 +46,7 @@ def run_tests(self):
     author_email=about['__author_email__'],
     description=about['__description__'],
     long_description=readme,
+    requires=['pyyaml'],
```
Comment (lk-geimfari) [inline setup.py]: Remove this, please, because `yaml` is optional feature.

**Comment by lk-geimfari** [comment]
@muhfajar Also, please add tests.

File: wasserstoff/utils.py (hunk 1 above)
Comment (muhfajar) [reply wasserstoff/utils.py]: Ok, I'll change PyYML as optional package

**Pull request closed without merging**